lib/active_merchant/billing/gateways/adyen.rb

```diff
@@ -459,6 +466,23 @@ def add_browser_info(browser_info, post)
           userAgent: browser_info[:user_agent]
         }
       end
+
+      def failure_response(initial_response)
```

Comment:
Since the failure message is hardcoded for the case of recurring transactions not being supported, do you think this method name should be more specific? (Or alternatively, the message could be a parameter).

Reply:
I went with `unsupported_failure_response` so that it's a bit more descriptive.